source/_posts/akr-replica-setup.md

```diff
@@ -0,0 +1,151 @@
+---
```

Comment:
keyword is not linked with website

Reply:
I have added the nayan website link at the bottom/end of the blog